### DISCUSSION TITLE
TIL - 22/03/14

### DISCUSSION THREAD

**Discussion by leeheefull** [selab-hs/selab#124] [Check TIL]
@leeheefull @DongGeon0908

**Comment by leeheefull** [comment]
### 작업사항
* 댓글 테스트 케이스 작성

### 이슈

**Comment by DongGeon0908** [comment]
### 작업사항
- rds 수정 작업 진행
- 서브모듈 리펙토링

### 이슈
- Board에 대한 정의 (Board의 확장성에 대한 고찰)
